linera-views/src/backends/scylla_db.rs

```diff
@@ -787,12 +870,21 @@ impl TestKeyValueStore for JournalingKeyValueStore<ScyllaDbStoreInternal> {
 
 /// The `ScyllaDbStore` composed type with metrics
 #[cfg(with_metrics)]
-pub type ScyllaDbStore =
-    MeteredStore<LruCachingStore<MeteredStore<JournalingKeyValueStore<ScyllaDbStoreInternal>>>>;
+pub type ScyllaDbStore = MeteredStore<
+    LruCachingStore<
+        MeteredStore<
+            ValueSplittingStore<MeteredStore<JournalingKeyValueStore<ScyllaDbStoreInternal>>>,
+        >,
+    >,
+>;
 
 /// The `ScyllaDbStore` composed type
 #[cfg(not(with_metrics))]
-pub type ScyllaDbStore = LruCachingStore<JournalingKeyValueStore<ScyllaDbStoreInternal>>;
+pub type ScyllaDbStore =
+    LruCachingStore<ValueSplittingStore<JournalingKeyValueStore<ScyllaDbStoreInternal>>>;
 
 /// The `ScyllaDbStoreConfig` input type
 pub type ScyllaDbStoreConfig = LruSplittingConfig<ScyllaDbStoreInternalConfig>;
+
+/// The combined error type for the `ScyllaDbStore`.
+pub type ScyllaDbStoreError = ValueSplittingError<ScyllaDbStoreInternalError>;
```

Comment:
Alternatively, this file could concentrate on what is currently called `ScyllaDbStoreInternal`, and all these definitions will be in another file, maybe `backends/mod.rs`.

Reply:
We could do that change (and also for DynamoDb, ScyllaDb).
But I would prefer in another PR.